src/styles/card.css

```diff
@@ -0,0 +1,97 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.page__card {
+  margin: auto;
+}
+
+.card__image {
+  height: 134px;
+  margin: 16px 3px 40px;
+}
+
+.card__description {
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+  margin-bottom: 4px;
+  color: var(--color-black);
+}
+
+.card__code {
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  margin-bottom: 16px;
+  color: var(--color-grey);
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.rating__reviews {
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: var(--color-black);
+}
+
+.card__price-box {
+  margin-bottom: 16px;
+}
+
+.price-box {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
same suggestion. you should use only 2 BEM blocks: card and stars, and keep the name convention